deso/block.go

```diff
@@ -944,6 +956,161 @@ func (node *Node) GetValidatorPKIDFromSubAccountIdentifier(subAccount *types.Sub
 	return lib.NewPKID(validatorPKIDBytes), nil
 }
 
+func (node *Node) getStakeOps(txn *lib.MsgDeSoTxn, utxoOps []*lib.UtxoOperation, numOps int) []*types.Operation {
+	if txn.TxnMeta.GetTxnType() != lib.TxnTypeStake {
+		return nil
+	}
+
+	var operations []*types.Operation
+	for _, utxoOp := range utxoOps {
+		if utxoOp.Type == lib.OperationTypeStake {
```

Comment:
So I think that you only need to do an OUTPUT here because the INPUT is already covered by getBalanceModelSpends right? If yes, just note that in a comment.